applications/ParticleMechanicsApplication/python_scripts/mpm_explicit_solver.py

```diff
@@ -55,6 +58,16 @@ def _CreateSolutionScheme(self):
         is_compressible = self.settings["compressible"].GetBool()
         grid_model_part.ProcessInfo.SetValue(KratosParticle.IS_COMPRESSIBLE, is_compressible)
 
+        # Check whether the partitioned quadrature mpm (PQMPM) is used
+        is_pqmpm = self.settings["is_pqmpm"].GetBool()
```

Comment:
Are you sure that you want to limit this towards explicit MPM, I suppose it should also work for implicit, right?

Reply:
This can be done but requires a few changes and some proper thought - the main thing is to ensure the material response is only calculated for the one 'master' material point and not n times for n subpoints. This is for efficiency and also since material responses might be non-linear and can't be re-combined via averaging. In explicit this is simple since the stress update is only called once per timestep, but if the implicit needs iterations then the stress is updated each iteration within the timestep. But I agree that adding this for implicit makes sense (since the added splitting cost is less compared to the solve time) - I propose we do this in a future PR